### ISSUE TITLE
[BUG] npm login and npm adduser both return a 400 error bad request - email already registered

### ISSUE THREAD

**Issue by MikeDepies** [npm/cli#2295]
### Current Behavior:
When trying to publish a package (first time) I'm running into this exception in the cli when attempting to login. I have created an account on npmjs and am trying to pair my cli. I get the following when using either command:
```
Username: ezekielbulver
Password:
Email: (this IS public) ezbulverism@gmail.com
npm ERR! code E400
npm ERR! 400 Bad Request - PUT https://registry.npmjs.org/-/user/org.couchdb.user:ezekielbulver - That email has already been registered
```

### Environment:
```
{
  'svelte-store-extensions': '1.0.0',
  npm: '6.14.9',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '36.0',
  http_parser: '2.9.3',
  icu: '65.1',
  llhttp: '2.0.4',
  modules: '72',
  napi: '5',
  nghttp2: '1.40.0',
  node: '12.16.2',
  openssl: '1.1.1e',
  tz: '2019c',
  unicode: '12.1',
  uv: '1.34.2',
  v8: '7.8.279.23-node.34',
  zlib: '1.2.11'
}
```

**Comment by ljharb** [comment]
When signing up, you use your email. When logging in, you use your password. The implication is that you already created your account.

**Comment by MikeDepies** [comment]
How do I connect cli npm to my account then? That is the issue I am confused with. I did log in with my password. 

I get the same error with both npm adduser and npm login.

**Comment by ljharb** [comment]
Have you tried entering your username and password, and not entering your email?

If that still doesn't work, i'd email support@npmjs.com.

**Comment by MikeDepies** [comment]
There is no option to not enter the email? Did you have reason to believe that there should be a different prompt? Your responses have been mildly confusing.

**Comment by ljharb** [comment]
@MikeDepies sorry if i've been confusing. What I mean is, when it prompts you for an email, just leave it blank and hit enter.

**Comment by MikeDepies** [comment]
it literally won't proceed without one.

**Comment by ljharb** [comment]
In that case, I’d try emailing npm support.

**Comment by paulsmithkc** [comment]
I have the same issue when trying to login on a second computer.

**Comment by DoronGeyer** [comment]
Hey @MikeDepies probably waay late to answer this , but I'll leave a possible answer for others that google this. 
When you create an npmjs account via terminal you can end up  with that error if your password is less that 10 characters long. 
If you use the actual npmjs.org website to create an account you will see that it gives you a notification of that issue. Try increase password length to 10+ characters in terminal and it should work .

Cheers, D.

**Comment by hochan222** [comment]
I keep a record for others who have the same cause as me. In my case, even though I entered the ID incorrectly, `npm ERR! 400 Bad Request - PUT https://registry.npmjs.org/-/user/org.couchdb.user:hochan222 - That email has already been registered` error message. There definitely seems to be a problem with the logic.

### TL;DR

**If the above error appears, check if the input values ​​other than the e-mail are correct.**

**Comment by TheCryptoMint** [comment]
This is still an issue. Frustrating as hell. Can we have a fix please?

**Comment by CastilloLuis** [comment]
I'm having the same issue atm. I'm trying to perform some npmjs action in my terminal but is not possible. I already changed account/pw twice and i'm still getting the issue while trying to log-in with `npm login`

**Comment by YongzeYao** [comment]
I encountered this as well. Any fix?

**Comment by pckrishnadas88** [comment]
for wrong usernames the error messages displayed as email already registered. Please double check the username entered from the web login page. It's a bug.

**Comment by SuhasParameshwara** [comment]
I'm still facing this issue. Any workaround?

**Comment by Xxs-7** [comment]
I encountered this problem recently. After double-checking my username, I realized that it was incorrect., By correcting my username, I was able to successfully log in. I suspect that the login process may involve both sign-in and sign-up.
